### DISCUSSION TITLE
q-table go back to first page after sorting

### DISCUSSION THREAD

**Discussion by makx6** [quasarframework/quasar#10337] [General - Components / Directives / etc] [answered]
If you go to the second page and sort it, it redirects to the first page. I want to stay on the same page where I did the sorting
Example: https://codepen.io/turigeza/pen/OGVwvz?&editable=true&editors=101

**Comment by ontwikkelfabriek** [comment, marked as answer]
@makx6 

Not easy to do since the sorting will put you on page 1 no matter what.( that code is in the Quasar component)
I would create a Feature Request for this.

Here's some hacky way that works(very buggy) for the first time you sort on the same page nr, the second time it will still go to page 1. You could enhance the code so it will work properly:
https://codepen.io/ontwikkelfabriek/pen/vYmvjoG

ah and you'll have to implement your **own** sorting function.( just copy it from the component source i guess)

**Reply by makx6** [reply]
Thank you
